src/views/sources/showSourceConnectionsModal.tsx

```diff
@@ -8,15 +8,15 @@ import React from 'react';
 import { Modal, ModalVariant, Button, Icon, List, ListItem } from '@patternfly/react-core';
 import { ExclamationCircleIcon, ExclamationTriangleIcon, CheckCircleIcon } from '@patternfly/react-icons';
 import { Tbody, Tr, Td, Table, ExpandableRowContent } from '@patternfly/react-table';
-import { SourceType, ConnectionType } from '../../types/types';
+import { type SourceType, type ConnectionType } from '../../types/types';
 import './showSourceConnectionsModal.css';
 
 export interface ConnectionsModalProps {
   source: SourceType;
   onClose: () => void;
   connections: {
     successful: ConnectionType[];
-    failure: ConnectionType[];
+    failed: ConnectionType[];
     unreachable: ConnectionType[];
   };
```

Comment:
future:
similar comment to a centralized type for `connections`... it's in a few places, maybe more